modules/ROOT/pages/gov-cloud-runtime-configuration.adoc

```diff
@@ -22,11 +24,9 @@ wrapper.java.additional.<n>=-Danypoint.platform.base_uri=https://gov.anypoint.mu
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.gov.anypoint.mulesoft.com
 ----
 
-In the previous example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
-
 === Command Line
 
-To configure Mule Runtime from the command line, use the following options when starting the Mule Runtime. Note that `./` only applies to Unix environments.
+To configure Mule runtime engine from the command line, use the following options when starting Mule runtime engine. Note that `./` only applies to Unix environments.
```

Comment:
I suggest not using the full name twice in one sentence, just for the "sound" of it:
```suggestion
To configure Mule runtime engine from the command line, use the following options when starting Mule. Note that `./` applies only in Unix environments.
```